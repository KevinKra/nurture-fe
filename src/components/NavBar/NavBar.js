import React from 'react';
import { Link } from 'react-router-dom';
import './NavBar.scss';

export default function NavBar(props) {
	return (
		<nav className="top-nav">
			<Link to="/home">Home</Link>
			<Link to="/register">Register</Link>
			<Link to="/login">Login</Link>
			<button onClick={props.handleLogout}>Logout</button>
		</nav>
	);
}
