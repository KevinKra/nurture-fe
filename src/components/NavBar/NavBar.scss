.top-nav {
	border: 1px solid red;
}
