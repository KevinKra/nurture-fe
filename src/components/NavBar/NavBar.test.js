import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';

describe('NavBar', () => {
	it('renders the correct navbar if the user is not logged in', () => {
		const wrapper = shallow(<NavBar />);
		expect(wrapper).toMatchSnapshot();
	});
	it('renders the correct navbar when the user logged in', () => {
		const wrapper = shallow(<NavBar loggedInStatus={'LOGGED_IN'} />);
		expect(wrapper).toMatchSnapshot();
	});
});
