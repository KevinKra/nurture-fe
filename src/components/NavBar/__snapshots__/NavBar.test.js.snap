// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders the correct navbar if the user is not logged in 1`] = `
<nav
  className="top-nav"
>
  <Link
    to="/"
  >
    Home
  </Link>
  <Link
    to="/register"
  >
    Register
  </Link>
  <Link
    to="/login"
  >
    Login
  </Link>
</nav>
`;

exports[`NavBar renders the correct navbar when the user logged in 1`] = `
<nav
  className="top-nav"
>
  <Link
    to="/"
  >
    Home
  </Link>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</nav>
`;
